asreview/webapp/api.py

```diff
@@ -250,32 +251,15 @@ def api_get_project_info(project):  # noqa: F401
 
     project_config = project.config
 
-    try:
-
-        # check if there is a dataset
-        try:
-            get_data_file_path(project.project_path)
-            project_config["projectHasDataset"] = True
-        except Exception:
-            project_config["projectHasDataset"] = False
```

Comment:
This `projectHasDataset` property is needed in the frontend.

Reply:
Yes, thanks. Please check the project info for `dataset_path`. If not exists or is null, then there is no dataset. Does that help?